ert_shared/models/base_run_model.py

```diff
@@ -80,8 +80,8 @@ def startSimulations(self, arguments):
             self.updateDetailedProgress()
             self.completed_realizations_mask = run_context.get_mask()
         except ErtRunError as e:
-            self.completed_realizations_mask = BoolVector(default_value=False)
             self.updateDetailedProgress()
+            self.completed_realizations_mask = BoolVector(default_value=False)
```

Comment:
This isn't strictly necessary, but the masking has an effect on updating detailed progress.